specification/redis/resource-manager/Microsoft.Cache/stable/2022-06-01/redis.json

```diff
@@ -2666,6 +2666,16 @@
             "name": "ReplicationRole",
             "modelAsString": false
           }
+        },
+        "geoReplicatedPrimaryHostName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Fully qualified geo subdomain URL which points to geo-primary cache among the linked redis caches."
+        },
+        "geoReplicatedPrimaryHostNameDnsCName": {
```

Comment:
This name 'geoReplicatedPrimaryHostNameDnsCName' doesn't seem clear in purpose to me. Also, it seems like it might be capturing unwanted implementation detail. Is it really important to expose to customers that this is a CNAME DNS record, or is it fine for us to just call it a DNS name?

Reply:
As I compare the descriptions of these two new properties, its not clear to me what 'actual' means, and its also not clear whether/how the customer is supposed to use these two properties differently.

Reply:
Simplified the name and updated description.